ocs_ci/ocs/resources/pod.py

```diff
@@ -3394,3 +3469,60 @@ def get_mon_pod_by_pvc_name(pvc_name: str):
         .get("items")[0]
     )
     return Pod(**mon_pod_ocp)
+
+
+def get_debug_pods(debug_nodes, namespace=constants.OPENSHIFT_STORAGE_NAMESPACE):
+    """
+    Get debug pods created for the nodes in debug
+
+    Args:
+        debug_nodes (list): List of nodes in debug mode
+        namespace (str): By default 'openshift-storage' namespace
+
+    Returns:
+        List: of Pod objects
+
+    """
+    debug_pods = []
+    for node_name in debug_nodes:
+        debug_pods.extend(
+            get_pod_obj(pod, namespace=namespace)
+            for pod in get_pod_name_by_pattern(
+                f"{node_name}-debug", namespace=namespace
+            )
+        )
+
+    return debug_pods
+
+
+def wait_for_pods_deletion(
+    label, timeout=120, sleep=5, namespace=constants.OPENSHIFT_STORAGE_NAMESPACE
+):
+    """
+    Wait for the pods with particular label to be deleted
+    until the given timeout
+
+    Args:
+        label (str): Pod label
+        timeout (int): Timeout
+        namespace (str): Namespace in which pods are running
+
+    Raises:
+        TimeoutExpiredError
+
+    """
+
+    def _check_if_pod_deleted(label, namespace):
+        if len(get_pods_having_label(label, namespace)) == 0:
+            return True
+        else:
+            return False
```

Comment:
you can simply return with the condition without the need to if
return len(get_pods_having_label(label, namespace)) == 0